File: 2026-2/sesion-de-clase-4/dotnet/Lab01-2026-1/AssessmentItem.cs
namespace Lab01_2026_1;

public class AssessmentItem
{
    public Question Pregunta { get; }
    public int Puntaje { get; }
    public bool EsCorrecto { get; set; }

    public AssessmentItem(Question pregunta, int puntaje)
    {
        Pregunta = pregunta;
        Puntaje = puntaje;
    }
}

File: 2026-2/sesion-de-clase-4/java/Lab01-2026-1/pe/edu/pucp/lab/examenes/AssessmentItem.java
package pe.edu.pucp.lab.examenes;

import pe.edu.pucp.lab.preguntas.Question;

public class AssessmentItem {

    private Question pregunta;
    private int puntaje;
    private boolean esCorrecto;

    public AssessmentItem(Question pregunta, int puntaje) {
        this.pregunta = pregunta;
        this.puntaje = puntaje;
    }

    public void setEsCorrecto(boolean esCorrecto) {
        this.esCorrecto = esCorrecto;
    }

    public Question getPregunta() {
        return pregunta;
    }

    public int getPuntaje() {
        return puntaje;
    }

    public boolean getEsCorrecto() {
        return esCorrecto;
    }
}
